### PR TITLE
Adds support for day threshold to the schema

### PR DESCRIPTION
**Pull request by orta**
Adds a new arg for https://github.com/artsy/gravity/pull/12218

The status is already there.

### CODE REVIEW

**Comment by ashkan18** [comment]
we need `status` too, right? https://github.com/artsy/gravity/pull/12218/files#diff-e7e7c00aa36f3a5f5580878cb1728ed8R14

**Comment by orta** [comment]
It's already in, we added that in the first run (see the schema diff )

File: src/schema/me/followed_shows.ts [hunk 1]
```diff
@@ -40,6 +40,11 @@ const FollowedShows: GraphQLFieldConfig<void, ResolverContext> = {
   args: pageable({
     status: EventStatus,
     sort: PartnerShowSorts,
+    dayThreshold: {
+      type: GraphQLString,
```
Comment (ashfurrow) [inline src/schema/me/followed_shows.ts]: This should be a `GraphQLInt`?

Comment (ashkan18) [reply src/schema/me/followed_shows.ts]: yes to ☝️

Comment (ashkan18) [reply src/schema/me/followed_shows.ts]: I can update this @orta

Comment (ashfurrow) [reply src/schema/me/followed_shows.ts]: He already did here: https://github.com/artsy/metaphysics/pull/1584

Comment (ashkan18) [reply src/schema/me/followed_shows.ts]: nvm looks to be already `int` on `master` :)

**Comment by alloy** [comment]
I think we’ve informally adopted a system where additions to the schema that simply pass-through data already resolved by a resolver doesn't warrant a test _as much_ as actual new logic. But perhaps a topic for the FE practice. /cc @zephraph

**Pull request merged**